e2e/case/camera-opaque-texture.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * @title Opaque Texture
+ * @category Camera
+ */
+import {
+  Animator,
+  BaseMaterial,
+  Camera,
+  DirectLight,
+  Engine,
+  Entity,
+  GLTFResource,
+  Logger,
+  MeshRenderer,
+  PrimitiveMesh,
+  RenderFace,
+  Shader,
+  Vector3,
+  WebGLEngine
+} from "@galacean/engine";
+import { OrbitControl } from "@galacean/engine-toolkit";
+import { initScreenshot, updateForE2E } from "./.mockForE2E";
+
+Logger.enable();
+WebGLEngine.create({ canvas: "canvas" }).then((engine) => {
+  engine.canvas.resizeByClientSize(2);
+  const scene = engine.sceneManager.activeScene;
+  const rootEntity = scene.createRootEntity();
+
+  // camera
+  const cameraEntity = rootEntity.createChild("camera_node");
+  cameraEntity.transform.position = new Vector3(0, 1, 3);
+  const camera = cameraEntity.addComponent(Camera);
+  cameraEntity.addComponent(OrbitControl).target = new Vector3(0, 1, 0);
+  camera.opaqueTextureEnabled = true;
+
+  const lightNode = rootEntity.createChild("light_node");
+  lightNode.addComponent(DirectLight).intensity = 0.6;
+  lightNode.transform.lookAt(new Vector3(0, 0, 1));
+  lightNode.transform.rotate(new Vector3(0, 90, 0));
+
+  engine.resourceManager
+    .load<GLTFResource>("https://gw.alipayobjects.com/os/bmw-prod/5e3c1e4e-496e-45f8-8e05-f89f2bd5e4a4.glb")
+    .then((gltfResource) => {
+      const { defaultSceneRoot } = gltfResource;
+      rootEntity.addChild(defaultSceneRoot);
+      const animator = defaultSceneRoot.getComponent(Animator);
+      animator.play("agree");
+
+      showDepthPlane(engine, cameraEntity);
+
+
+      updateForE2E(engine);
+
+      initScreenshot(engine, camera);
+    });
+});
+
+function showDepthPlane(engine: Engine, camera: Entity): void {
+  const entity = camera.createChild("Plane");
+  entity.transform.setPosition(0, 0, -1);
+  entity.transform.rotate(new Vector3(-90, 0, 0));
```

Comment:
rotate 90 and close `doubleFace` to test `flipY`

Reply:
done